### PR TITLE
update developer guide to correct file path

### PR DESCRIPTION
**Pull request by jinchihe**
**What this PR does / why we need it**:

Seems the sample file `docs/samples/tensorflow.yaml` in developer guide has been removed by PR #118 , the PR is to correct file path to point the `tensorflow` folder sample so that the developer guide is performable. Thanks.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/124)

### CODE REVIEW

File: docs/DEVELOPER_GUIDE.md [hunk 1]
```diff
@@ -153,12 +153,12 @@ and deploys the `kfserving-controller-manager` with the image digest to your clu
...
-kubectl apply -f docs/samples/tensorflow.yaml
+kubectl apply -f docs/samples/tensorflow/tensorflow.yaml
 ```
 You should see model serving deployment running under default or your specified namespace.
 
 ```console
-$ kubectl get pods -n default -l serving.kubeflow.org/kfservice=my-model
+$ kubectl get pods -n default -l serving.kubeflow.org/kfservice=flowers-sample
 NAME                                                READY   STATUS    RESTARTS   AGE
 my-model-default-htz8r-deployment-8fd979f9b-w2qbv   3/3     Running   0          10s
```
Comment (yuzisun) [inline docs/DEVELOPER_GUIDE.md]: Can you help change `my-model-default` to `flowers-sample-default` ?

Comment (jinchihe) [reply docs/DEVELOPER_GUIDE.md]: Oh... That's great catch! :-)

**Comment by yuzisun** [comment]
Thanks for fixing! 

/lgtm
/approve

**Pull request merged**